src/Storages/System/StorageSystemServerSettings.cpp

```diff
@@ -18,6 +19,24 @@ namespace CurrentMetrics
 
 namespace DB
 {
+
+static std::vector<std::pair<String, Int8>> getTypeEnumsAndValues()
+{
+    return std::vector<std::pair<String, Int8>>{
+        {"Full",            static_cast<Int8>(RuntimeReloadType::FULL)},
```

Comment:
```cpp
        {"Yes",             static_cast<Int8>(RuntimeReloadType::Yes)},
        {"OnlyIncrease",    static_cast<Int8>(RuntimeReloadType::OnlyIncrease)},
        {"OnlyDecrease",    static_cast<Int8>(RuntimeReloadType::OnlyDecrease)},
        {"No",              static_cast<Int8>(RuntimeReloadType::No)},
```

Reply:
@rschu1ze Hi, Perhaps the enumeration should be all capitalized and separated by underscores?

Reply:
My proposal was mostly of stylistic kind: `ALL_CAPITALIZATION_WITH_UNDERSCORES` is mostly used by C-style macros, in the C++ world, `CamelCaseWithoutUnderscores` is preferred.